modules/social_features/social_user/social_user.tokens.inc

```diff
@@ -36,21 +35,14 @@ function social_user_token_info() {
 function social_user_tokens($type, $tokens, array $data, array $options, BubbleableMetadata $bubbleable_metadata) {
   $replacements = [];
 
-  if ($type == 'social_user' && !empty($data['message'])) {
-    /** @var \Drupal\message\Entity\Message $message */
-    $message = $data['message'];
```

Comment:
@nkoporec @robertragas As I understood from the description of PR that we pass $data['message'] as a string, not Message entity. But $data['message'] should always be "Message" entity.

Did you observed why this was sent as "string" ?

For example, please see: https://git.drupalcode.org/project/message/-/blob/8.x-1.2/message.tokens.inc#L56